grakn-graph/src/main/java/ai/grakn/graph/internal/AbstractGraknGraph.java

```diff
@@ -136,6 +137,13 @@ public AbstractGraknGraph(G graph, String keyspace, String engine, boolean batch
     }
 
     /**
+     * Opens the thread bound transaction
+     */
+    public void openTransaction(){
+        localIsOpen.set(true);
+    }
```

Comment:
Does this do the same as what `open` used to do? I still find this behaviour a little confusing, but the name is better.